app/__init__.py

```diff
@@ -44,3 +45,4 @@
 api.add_namespace(image_ocr_ns, path='/image/ocr')
 api.add_namespace(article_ns, path='/article')
 api.add_namespace(graph_ns, path='/graph/cluster')
+api.add_namespace(presto_ns, path='/presto')
```

Comment:
This is the callback endpoint that Presto calls?

Reply:
Yep!